src/Compiler/Errors/Errors.ts

```diff
@@ -38,40 +38,40 @@ export const enum BriskErrorType {
 // Error Strings
 export const BriskErrorMessage = {
   // Brisk Error
-  [BriskErrorType.CompilerError]: 'Compiler Bug Please Report',
-  [BriskErrorType.FeatureNotYetImplemented]: 'Feature Not Yet Implemented',
-  [BriskErrorType.ImportNotSupported]: 'Your Current Environment Does Not Support Imports',
+  [BriskErrorType.CompilerError]: 'A compiler bug has occurred',
+  [BriskErrorType.FeatureNotYetImplemented]: 'Not implemented',
+  [BriskErrorType.ImportNotSupported]: 'Your environment does not support imports',
   // Syntax Error
-  [BriskErrorType.CannotUseReservedKeyword]: 'Cannot Use Reserved Keyword %1',
+  [BriskErrorType.CannotUseReservedKeyword]: 'Cannot use reserved keyword `%1`',
   [BriskErrorType.ReturnStatementsOnlyValidInsideFunction]:
-    'Return Statements Only Valid Inside Function',
+    'Return statements must be inside a function',
   [BriskErrorType.InvalidNumber]: 'Number `%1` Is Invalid',
```

Comment:
Missed One

Reply:
What is an invalid number ?????? This error is not even used anywhere ??????